core/src/test/java/be/orbinson/aem/dictionarytranslator/servlets/action/ImportDictionaryServletTest.java

```diff
@@ -0,0 +1,106 @@
+package be.orbinson.aem.dictionarytranslator.servlets.action;
+
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.testing.mock.sling.servlet.MockSlingHttpServletRequest;
+import org.apache.sling.testing.mock.sling.servlet.MockSlingHttpServletResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(AemContextExtension.class)
+class ImportDictionaryServletTest {
+
+    private ImportDictionaryServlet importTranslation;
+
+    private AemContext context;
+
+    @BeforeEach
+    void setUp(final AemContext ctx) {
+        context = ctx;
+        importTranslation = context.registerInjectActivateService(new ImportDictionaryServlet());
+        context.create().resource("/content");
+    }
+
+    @Test
+    void testDoPost_successful() throws Exception {
+        MockSlingHttpServletRequest request = context.request();
+        MockSlingHttpServletResponse response = context.response();
+
+        // Set request parameters
+        request.addRequestParameter("dictionary", "/content");
+        String csvData = "Labelname,EN,NL\nhello,Hello,Hallo\n";
+        InputStream csvStream = new ByteArrayInputStream(csvData.getBytes());
+        request.addRequestParameter("./csvfile", csvStream.readAllBytes(), "text/csv", "translations.csv");
+
+        // Call servlet's doPost method
```

Comment:
comments not needed, no added value other than repeat what the method implies

Reply:
update your ChatGPT prompt when generating tests :D

Reply:
removed comments